demo-java/app/src/gms/java/com/example/placesdemo/AutocompleteAddressActivity.java

```diff
@@ -131,15 +151,27 @@ public void onMapReady(GoogleMap googleMap) {
                     MapStyleOptions.loadRawResourceStyle(this, R.raw.style_json));
 
             if (!success) {
-                Log.e(TAG, "Style parsing failed.");
+                e(TAG, "Style parsing failed.");
             }
         } catch (Resources.NotFoundException e) {
-            Log.e(TAG, "Can't find style. Error: ", e);
+            e(TAG, "Can't find style. Error: ", e);
         }
         map.moveCamera(CameraUpdateFactory.newLatLngZoom(coordinates, 15f));
         marker = map.addMarker(new MarkerOptions().position(coordinates));
     }
 
+    public void onCheckboxClicked(View view) {
```

Comment:
Would this listener still work if you change the signature to?

```
public void onCheckboxClicked(CheckBox view)
```

Reply:
It seems to fail. Using the CompoundButton parent class setOnCheckedChangeListener instead.